### PR TITLE
Fix translation org

### PR DESCRIPTION
**Pull request by DjamelSoualmi**
[Ticket](https://trello.com/c/erE2lU5K/2400-organism-traduction-vertical-tab)

tran-org

**Detailed purpose of the PR**

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/organism/playlist-translation/style.css [hunk 1]
```diff
@@ -9,13 +9,19 @@
 .container {
   display: flex;
   background-color: cm_grey_50;
+  position: relative;
 }
 
 .menu {
   background-color: cm_grey_50;
-  height: 360px;
+  height: 88%;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/organism/playlist-translation/style.css:17]:
```suggestion
  height: auto;
```

**Review by esa-coorp — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/organism/playlist-translation/style.css (hunk 1 above)
Comment (DjamelSoualmi) [reply packages/@coorpacademy-components/src/organism/playlist-translation/style.css]: done

**Review by esa-coorp — APPROVED** [review]

File: packages/@coorpacademy-components/src/organism/playlist-translation/style.css [hunk 2]
```diff
@@ -9,13 +9,19 @@
...
 
 .menu {
   background-color: cm_grey_50;
-  height: 360px;
+  height: auto;
   width: 200px;
   margin: 24px 8px 24px 24px;
+  position: absolute;
+  top: 0px;
+  left: 0px;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/organism/playlist-translation/style.css:22]:
```suggestion
  top: 0;
  left: 0;
```

File: packages/@coorpacademy-components/src/organism/playlist-translation/style.css [hunk 3]
```diff
@@ -9,13 +9,19 @@
...
+  z-index: 1;
+  bottom: 0px;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/organism/playlist-translation/style.css:24]:
```suggestion
  bottom: 0;
```

**Review by Fausto95 — APPROVED** [review]

**Pull request merged**